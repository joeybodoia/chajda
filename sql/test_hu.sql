CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('hu',' "Ez a példakénti tesztmondat');
