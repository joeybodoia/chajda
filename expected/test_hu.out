CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
       spacy_lemmatize        
------------------------------
   példakénti:4 tesztmondat:5
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
     spacy_lemmatize      
--------------------------
   példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
            spacy_lemmatize            
---------------------------------------
   ez:2 a:3 példakénti:4 tesztmondat:5
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
        spacy_lemmatize        
-------------------------------
   ez a példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
         spacy_lemmatize          
----------------------------------
   ":2 példakénti:5 tesztmondat:6
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
      spacy_lemmatize       
----------------------------
   " példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
              spacy_lemmatize              
-------------------------------------------
   ":2 ez:3 a:4 példakénti:5 tesztmondat:6
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
         spacy_lemmatize         
---------------------------------
   " ez a példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
       spacy_lemmatize        
------------------------------
   példakénti:4 tesztmondat:5
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
     spacy_lemmatize      
--------------------------
   példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
            spacy_lemmatize            
---------------------------------------
   Ez:2 a:3 példakénti:4 tesztmondat:5
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
        spacy_lemmatize        
-------------------------------
   Ez a példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
         spacy_lemmatize          
----------------------------------
   ":2 példakénti:5 tesztmondat:6
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
      spacy_lemmatize       
----------------------------
   " példakénti tesztmondat
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
              spacy_lemmatize              
-------------------------------------------
   ":2 Ez:3 a:4 példakénti:5 tesztmondat:6
(1 row)

SELECT spacy_lemmatize('hu',' "Ez a példakénti tesztmondat' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
         spacy_lemmatize         
---------------------------------
   " Ez a példakénti tesztmondat
(1 row)

SELECT spacy_tsvector('hu',' "Ez a példakénti tesztmondat');
         spacy_tsvector         
--------------------------------
 'példakénti':4 'tesztmondat':5
(1 row)

